hook-common/src/pgqueue.rs

```diff
@@ -560,9 +688,8 @@ impl PgQueue {
         let name = queue_name.to_owned();
         let table = table_name.to_owned();
         let pool = PgPoolOptions::new()
-            .connect(url)
-            .await
-            .map_err(|error| PgQueueError::ConnectionError { error })?;
+            .connect_lazy(url)
```

Comment:
As far as switching to `connect_lazy`: my gut feeing is that PG can be down any time, and we shouldn't crash-loop on startup if it's unavailable. It seems better to always present the same API to users, if PG is unavailable maybe we 500, but eventually things will work again.

Reply:
Yeah, fair enough we can connect later :+1: